ocdskingfisherprocess/transform/compile_releases.py

```diff
@@ -0,0 +1,61 @@
+from ocdskingfisherprocess.transform.base import BaseTransform
+import sqlalchemy as sa
+import ocdsmerge
+
+
+class CompileReleaseTransform(BaseTransform):
+    type = 'COMPILED-RELEASES'
```

Comment:
> Will all transforms have a unique type?

Yes

> If so, why not just use the class name as the way of identifying the transform, instead of type?

I just thought it was good to have something different from the class name, so we have freedom to change class names or anything to do with the code structure in the future and there is a level of abstraction there. Also it can be a clearer read for people looking at the DB directly.

Reply:
Makes sense – I might prefer an all-lowercase `type` (easier to type).